erpnext/accounts/deferred_revenue.py

```diff
@@ -169,6 +175,28 @@ def _book_deferred_revenue_or_expense(item):
 		if item.get(enable_check):
 			_book_deferred_revenue_or_expense(item)
 
+def process_deferred_accounting_record(posting_date=today()):
+	''' Converts deferred income/expense into income/expense
+		Executed via background jobs on every month end '''
+
+	if not cint(frappe.db.get_singles_value('Accounts Settings', 'automatically_process_deferred_accounting_entry')):
```

Comment:
Since the old system used to process deferred entries automatically without this setting check this setting in a patch

Reply:
set the option by default